### ISSUE TITLE
problem malych oken (resp modre cary)

### ISSUE THREAD

**Issue by evapytlounova** [ParalelniPolis/hcpp2015main#47]
kdyz jdes na web, tak ta modra cara na konci prvni sekce se vzdycky zobrazi v okne uplne dole - tedy kdyz neni okno dost vysoke, ustreli ten odpocet (at uz uplne, nebo castecne)

po scrollu dolu to pak vypada takhle:

![run in small window](https://cloud.githubusercontent.com/assets/14018808/9569991/b534a3b0-4f7c-11e5-8860-4fd5ea53c1c8.png)

**Comment by tomtomklima** [comment]
Asi bych neřešil, moc velká práce na malý výsledek. Někdo proti?

**Comment by evapytlounova** [comment]
tyjo, mne asi prijde, ze pokud bysme meli neco opravit, je to tohle...

**Comment by tomtomklima** [comment]
Nu OK - napadá mě zvětšit 100% na kupříkladu 130% při velikosti okna menší než treshold. Pokud budu mít o víkendu nějaký čas, zkusím to lousknout

**Comment by tomtomklima** [comment]
No; řešil jsem to asi půl hodiny, našel jsem jediné řešení bez JS: http://stackoverflow.com/a/14911949/4394743 a do toho se mi už dneska vážně nechce. Takže buď se toho někdo ujme a nebo zavírám. Btw; problém nastává jen když je nějak netypicky nasizeované okno, typicky dvakrát tak menší výška než šířka, takže common userům to asi ani tak nepřijde
